services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/ApplicationConfig.java

```diff
@@ -335,7 +377,7 @@ public HttpEndpoint deviceHttpEndpoint() {
         // need to use backend service because creating a device implicitly
         // creates (empty) credentials as well for the device
         final MongoDbBasedDeviceBackend service = new MongoDbBasedDeviceBackend(registrationService(),
-                credentialsService());
+                                                                                credentialsService());
```

Comment:
Could you please revert it back to the original version? The previous formatting was fine.

Reply:
reverted